src/main/java/org/apache/accumulo/testing/continuous/ContinuousBatchWalker.java

```diff
@@ -39,34 +40,33 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.collect.Sets;
+
 public class ContinuousBatchWalker {
   private static final Logger log = LoggerFactory.getLogger(ContinuousBatchWalker.class);
 
   public static void main(String[] args) throws Exception {
 
     try (ContinuousEnv env = new ContinuousEnv(args)) {
-      Authorizations auths = env.getRandomAuthorizations();
-      AccumuloClient client = env.getAccumuloClient();
-      Scanner scanner = ContinuousUtil.createScanner(client, env.getAccumuloTableName(), auths);
-      int scanBatchSize = Integer.parseInt(env.getTestProperty(TestProps.CI_BW_BATCH_SIZE));
-      scanner.setBatchSize(scanBatchSize);
-
-      Random r = new Random();
-
-      while (true) {
-        BatchScanner bs = client.createBatchScanner(env.getAccumuloTableName(), auths);
 
-        Set<Text> batch = getBatch(scanner, env.getRowMin(), env.getRowMax(), scanBatchSize, r);
-        List<Range> ranges = new ArrayList<>(batch.size());
+      Authorizations auths = env.getRandomAuthorizations();
 
-        for (Text row : batch) {
-          ranges.add(new Range(row));
+      try (AccumuloClient client = env.getAccumuloClient();
```

Comment:
Does the client get reused? I am wondering since it gets pulled from the `env`. I don't think it does, its just a convenience method...

Reply:
OK its probably not a big deal then but I'd remove the try-with-resources around the client since its redundant.

Reply:
Addressed these in [8a658d6](https://github.com/apache/accumulo-testing/pull/212/commits/8a658d64fe381d511292c24145cd46602d5bf66f)